src/views/Profile.js

```diff
@@ -426,7 +488,12 @@ const ProfileTrustButton = ({ address, trustStatus }) => {
           {trustStatus.isPending ? (
             <CircularProgress size={24} />
           ) : (
-            <TrustIcon className={classes.trustButtonIcon} />
+            <Box className={classes.trustButtonContainer}>
+              <IconTrustCustomShape
+                className={classes.trustButtonIconBackground}
+              />
+              <TrustIcon className={classes.trustButtonIcon} />
+            </Box>
```

Comment:
I think it would make sense to make this new Uneven Round Button into its own component.

430 lines for one component makes no sense to me. It should be easy to add such a button to another view than Profile.js. Such as the send button in SendConfirm.js for example.